outlierdetect/outlierdetect.py

```diff
@@ -323,9 +327,11 @@ def _run_alg(data, agg_col, cat_cols, model, null_responses):
         for agg_unit in agg_units:
             frequencies[agg_unit],grouped = _get_frequencies(data, col, col_vals, agg_col, agg_unit, agg_to_data)
             agg_col_to_data[agg_unit][col] = grouped
-        outlier_scores_for_col = model.compute_outlier_scores(frequencies)
+        outlier_scores_for_col, expected_frequencies_for_col = model.compute_outlier_scores(frequencies)
         for agg_unit in agg_units:
-            outlier_scores[agg_unit][col] = outlier_scores_for_col[agg_unit]
+            outlier_scores[agg_unit][col] = {'score': outlier_scores_for_col[agg_unit],
+                                             'observed_freq': frequencies[agg_unit],
+                                             'expected_freq': expected_frequencies_for_col[agg_unit]}
```

Comment:
Definitely need to update the documentation if you're making a breaking change in the API like this. The end result is no longer a score, but a dictionary with very specific keys, right?

Reply:
Correct! 

Besides the comments for the functions, where else do you suggest documenting the new output?

Reply:
I wonder if we should improve the README file. Feels like future work